pom.xml

```diff
@@ -110,13 +110,19 @@
     <dependency>
       <groupId>io.rest-assured</groupId>
       <artifactId>rest-assured</artifactId>
-      <version>${rest-assured.version}</version>
+      <version>4.3.1</version>
       <scope>test</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>com.sun.xml.bind</groupId>
+          <artifactId>jaxb-osgi</artifactId>
+        </exclusion>
+      </exclusions>
     </dependency>
     <dependency>
       <groupId>org.folio</groupId>
       <artifactId>mod-pubsub-client</artifactId>
-      <version>1.3.1</version>
+      <version>1.3.3</version>
```

Comment:
I thought this work was blocked by an upgrade of the `mod-pubsub-client` that supports vert.x 4.x. This version does not seem to do that. Was I mistaken?

Reply:
I suspected that version currently set for pubsub client, which is 1.3.3, doesn't contains recently merged changes. But I just wondering can be 1.4.0-SNAPSHOT set for now or only remains is to wait until new release of module will be emitted? I have concerns regarding timing of dependent module release, except the case when we asking folijet team to force module release to resolve dependency issues. What do you think, what approach is better in this situation? By the way, tests were passed because it looks I had disabled test case uses pubsub client, will revert it back to test suite.

Reply:
> But I just wondering can be 1.4.0-SNAPSHOT set for now or only remains is to wait until new release of module will be emitted? I have concerns regarding timing of dependent module release, except the case when we asking folijet team to force module release to resolve dependency issues. What do you think, what approach is better in this situation? 

Personally, I would prefer not to use `SNAPSHOT` dependencies because they create non-repeatable builds. 

It was my understanding that Folijet had agreed to release it early to mitigate this challenge.

Please do try the `SNAPSHOT` dependency to check if it works. I don't think this pull request should be merged until there is a non-snapshot artefact to depend upon.

> By the way, tests were passed because it looks I had disabled test case uses pubsub client, will revert it back to test suite.

Thank you for re-including those tests

Reply:
Thank you Marc for sharing your point of view for this question. I just have discussed this point with team responsible for pubsub client too, they planning to prepare release of version 1.4 this sprint. So for now I'll leave SNAPSHOT there and will update dependency once new version be released.

Reply:
I have updated the version of dependency of pubsub-client to latest released one - 2.0.0.